dts/bindings/lora/semtech,sx126x-base.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2019 Manivannan Sadhasivam
+# Copyright (c) 2020 Andreas Sandberg
+# SPDX-License-Identifier: Apache-2.0
+
+include: spi-device.yaml
+
+properties:
+    reset-gpios:
+      type: phandle-array
+      required: true
+      description: |
+        GPIO connected to the sensor NRST signal.
+
+        This signal is open-drain, active-low as interpreted by the
+        modem.
+
+    busy-gpios:
+      type: phandle-array
+      required: true
+      description: |
+        GPIO connected to the sensor BUSY signal.
```

Comment:
s/sensor/modem/ ?

Reply:
Weird. Must have been a copy-and-paste error. It should be fixed now.